international-standard/document.adoc

```diff
@@ -1,35 +1,134 @@
 = ISO Codes for timekeeping systems
+:comment: ### See comment 1 below
 :docnumber: 99999
+:comment: ### Document number assigned by the Technical committee. See comment 1
 :tc-document-number: 9999
+:comment: ### Document edition
 :edition: 1
+:comment: ### The year of copyright
 :copyright-year: 2029
-:language: en
+:comment: ### Introductory component of the English title of the document. 
 :title-intro-en: Date and time
+:comment: ### Main component of the English title of the document (mandatory)
 :title-main-en: Codes for timekeeping systems
+:comment: ### Introductory component of the French title of the document.
 :title-intro-fr: Date et l'heure
+:comment: ###  Main component of the French title of the document (mandatory)
 :title-main-fr: Codes pour les systèmes de chronométrage
-:doctype: standard
+:comment: ### See comment 2 below
+:doctype: international-standard
+:comment: ### See comment 3 below
 :docstage: 00
+:comment: ### See comment 4 below
 :docsubstage: 00
+:comment: ### See comment 5 below
 :technical-committee-type: TC
+:comment: ### Number of the relevant ISO subcommittee
 :technical-committee-number: 154
+:comment: ### Name of the relevant ISO technical committee (mandatory)
 :technical-committee: Processes, data elements and documents in commerce, industry and administration
+:comment: ### See comment 6 below
 :workgroup-type: WG
+:comment: ### Number of the relevant ISO workgroup
 :workgroup-number: 5
+:comment: ### Name of the relevant ISO workgroup
 :workgroup: Date and time
+:comment: ### Secretariat example values: ANSI, SAC, ACA...
 :secretariat: SAC
+:comment: ### Sets document language. Two values are permitted: en, fr.
 :language: en
+:comment: ### Draft status, enables display of reviewer notes; remove if final document
 :draft:
+:comment: ### Enables table of contents
 :toc:
+:comment: ### Enables stem math mode
 :stem:
+:comment: ### Sets crossreference style
 :xrefstyle: short
+:comment: ### Sets images folder directory
 :imagesdir: images
+:comment: ### name of this AsciiDoc file
 :docfile: document.adoc
+:comment: ### Document class type
 :mn-document-class: iso
 :mn-output-extensions: xml,html,doc,html_alt
 :local-cache-only:
 :data-uri-image:
 
+
+////
+Comment 1: DOCUMENT NUMBER (:docnumber:)
+
+Sequence of digits that gives the document number value included in the document identifier, for example: 
+
+99999 (to author ISO-99999)
+8000 (to author ISO-8000), 
+22745 (to author ISO-22745), 
+29002 (to author ISO-29002)
+////
+
+////
+Comment 2: DOCUMENT TYPE (:doctype:) (mandatory)
```

Comment:
Also point them to the MN ISO page that describes these enums. From the ISO page, they would not be able to know what the actual values are.